src/main/java/com/exasol/releasedroid/adapter/github/GitHubReleaseMaker.java

```diff
@@ -48,8 +48,8 @@ private String createGitHubRelease(final Repository repository) {
 
     private GitHubRelease createReleaseModel(final Repository repository, final String version) {
         final ReleaseLetter releaseLetter = repository.getReleaseLetter(version);
+        final String header = version + releaseLetter.getHeader().map(h -> ": " + h).orElse("");
```

Comment:
General question, should not have the release letter have always a header? If so, we should not attach the empty string, if it is not available, but instead fail.

Reply:
Hi @tkilias I will check.

Reply:
I can understand your request and share your opinion. 
[design.md](https://github.com/exasol/release-droid/blob/main/doc/design.md) only says 

> RD extracts the GitHub release's title from the release letter.

Do we need to update or detail the requirements first?

Reply:
@redcatbear : 
> Codename is  mandatory.